extern/bullet2/src/BulletDynamics/Vehicle/btRaycastVehicle.h

```diff
@@ -61,7 +61,7 @@ class btRaycastVehicle : public btActionInterface
 	btVehicleRaycaster*	m_vehicleRaycaster;
 	btScalar		m_pitchControl;
 	btScalar	m_steeringValue; 
-	btScalar m_currentVehicleSpeedKmHour;
+	btScalar m_currentVehicleSpeedMetersPerSec;
```

Comment:
@DCubix : You should add in the bullet2 directory a patch file containing these changes: it's for futur upgrade to reapply patches.